src/app/public/modules/dropdown/dropdown.component.spec.ts

```diff
@@ -305,6 +315,23 @@ describe('Dropdown component', () => {
       expect(popoverTriggerType).toEqual('mouseenter');
     });
 
+    it('should constrain overflow and not be full screen when there are a long list of dropdown items', fakeAsync(() => {
+
+      component.setItems([
+        // tslint:disable-next-line:max-line-length
+        { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }, { name: 'Foo', disabled: false }
```

Comment:
Could we generate this array using a `for` loop or something? It's fairly difficult to read.

Reply:
Sure thing. Done.